packages/layout/components/App/App.tsx

```diff
@@ -18,9 +18,6 @@ const App = (props: AppProps) => {
         <SlotRenderer name="header" content={content} location={location} />
       </header>
       <SlotRenderer name="main" content={content} location={location} />
-      <footer id="footer">
```

Comment:
I'd bring back the footer in here, since it's valuable for the `App` component itself, in case somebody wants to use it anyways. It serve its purpose in Storybook story too. To make it more obvious: we won't be using it in Seven, since the Route outlet needs to be rendered in the middle (content).